roles/os_must_gather/tasks/main.yml

```diff
@@ -63,12 +63,26 @@
           oc adm must-gather --image {{ cifmw_os_must_gather_image }}
           --timeout {{ cifmw_os_must_gather_timeout }}
           --host-network={{ cifmw_os_must_gather_host_network }}
-          --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs/openstack-k8s-operators-openstack-must-gather
+          --dest-dir {{ cifmw_os_must_gather_output_log_dir }}
           -- ADDITIONAL_NAMESPACES={{ cifmw_os_must_gather_additional_namespaces }}
           OPENSTACK_DATABASES=$OPENSTACK_DATABASES
           SOS_EDPM=$SOS_EDPM
           SOS_DECOMPRESS=$SOS_DECOMPRESS
-          gather &> {{ cifmw_os_must_gather_output_dir }}/logs/openstack-k8s-operators-openstack-must-gather/os_must_gather.log
+          gather &> {{ cifmw_os_must_gather_output_log_dir }}/os_must_gather.log
+
+
+    - name: Find gather current dir
+      ansible.builtin.find:
+        paths: "{{ cifmw_os_must_gather_output_log_dir }}"
+        file_type: directory
+        depth: 1
+      register: _os_gather_latest_dir
+
+    - name: Make a symlink of os must gather log dir
+      ansible.builtin.file:
+        src: "{{ cifmw_os_must_gather_output_log_dir }}/{{ (_os_gather_latest_dir.files | sort(attribute='mtime', reverse=True) | first).path | basename }}"
```

Comment:
I think will be just `{{ (_os_gather_latest_dir.files | sort(attribute='mtime', reverse=True) | first).path | basename }}` ~ we would have to use relative path to the created `latest` symlink ~ otherwise on logs server the preserver absolute path that you set will be incorrect